source/jormungandr/jormungandr/scenarios/helper_classes/fallback_durations.py

```diff
@@ -183,13 +209,26 @@ def _do_request(self):
             or not len(sn_routing_matrix.rows[0].routing_response)
         ):
             logger.debug("no fallback durations found from %s by %s", self._requested_place_obj.uri, self._mode)
-            return result
+            return result, orig_to_parking_durations
 
         for pos, r in enumerate(sn_routing_matrix.rows[0].routing_response):
             if r.routing_status != response_pb2.unreached:
                 duration = self._get_duration(r, places_isochrone[pos])
-                if duration < self._max_duration_to_pt:
-                    result.update({places_isochrone[pos].uri: DurationElement(duration, r.routing_status)})
+                if self._mode == 'car':
+                    for sp_nearby in places_isochrone[pos].stop_points_nearby:
+                        duration_to_stop_point = self._get_manhattan_duration(
+                            sp_nearby.distance, self._speed_switcher.get('walking')
+                        )
+                        if (duration + duration_to_stop_point) < self._max_duration_to_pt:
+                            result[sp_nearby.uri] = DurationElement(
```

Comment:
I need your opinion @xlqian, It is the good way to do `duration = origin_to_Parking_duration + parking_to_Stop_point_duration` for the result ?